documentation/src/main/asciidoc/chapter-10-advanced-mapping-options.asciidoc

```diff
@@ -364,6 +366,11 @@ If there is a custom `@Condition` method applicable for the property it will hav
 Methods annotated with `@Condition` in addition to the value of the source property can also have the source parameter as an input.
 ====
 
+[NOTE]
+====
+`nullValuePropertyMappingStrategy` will influence what to do when the condition is not met.
```

Comment:
This is only valid for update mappings and not for other normal mappings. We have an entire section about [Controlling mapping result for null properties](https://mapstruct.org/documentation/stable/reference/html/#mapping-result-for-null-properties) that talks about these things. So perhaps we need to link to that.

Reply:
Correct, I missed that bit. Will rewrite this as well as the above.